readthedocs/templates/docsitalia/overrides/builds/build_detail.html

```diff
@@ -0,0 +1,244 @@
+            {% endblocktrans %}
+          </p>
+        </div>
+      {% endif %}
+    {% endif %}
+
+    {% if build.output %}
+      {# If we have build output, this is an old build #}
+      <p>
+        <button data-bind="visible: !legacy_output(), click: show_legacy_output">
+          {% trans "Show full build output" %}
+        </button>
+      </p>
+
+      <div data-bind="visible: legacy_output"
+           style="display: none;">
+        <h3>{% trans "Build Output" %}</h3>
+        <pre class="build-output"><span id="build-output">{{ build.output }}</span></pre>
+
+        {% if build.setup %}
+          <h3>{% trans "Setup Output" %}</h3>
+          <pre class="build-output"><span id="build-setup">{{ build.setup }}</span></pre>
+        {% endif %}
+
+        {% if build.setup_error %}
+          <h3>{% trans "Environment Standard Error" %}</h3>
+          <pre class="build-error"><span id="build-setup_error">{{ build.setup_error }}</span></pre>
+        {% endif %}
+      </div>
+
+      {% if build.error %}
+        <h3>{% trans "Build Errors" %}</h3>
+        <pre class="build-error"><span id="build-error">{{ build.error }}</span></pre>
+      {% endif %}
+
+    {% else %}
+      {# Show new command output if lacking build.output #}
+
+      <div class="build-error"
+           data-bind="visible: error"
+           style="display: none;">
+        <h3>{% trans "Error" %}</h3>
+        <p class="build-error"
+           data-bind="text: error">
+          {{ build.error }}
+        </p>
+        <p>
+          {% block github_issue_link %}
+            {% if issue_url %}
+              {% blocktrans trimmed with url=issue_url %}
+                <a href="{{ url }}">Report any build issues here</a>.
+              {% endblocktrans %}
+            {% endif %}
+          {% endblock %}
+        </p>
+      </div>
+
+      <div id="build-commands"
+           class="build-command-list"
+           data-bind="visible: commands, foreach: display_commands"
+           style="display: none;">
+        <div class="build-command"
+             data-bind="css: command_status">
+          <div class="build-command-run" data-bind="click: toggleCommand">
+            <span data-bind="text: command"></span>
+          </div>
+          <div class="build-command-return"
+               data-bind="visible: is_showing">
+            <div class="build-command-output">
+              <span data-bind="text: output"></span>
+            </div>
+
+            <div class="build-command-meta">
+              <span class="build-command-time">
+                {% trans "Command time:" %}
+                <span data-bind="text: run_time">0</span>s
+              </span>
+
+              <span class="build-command-exit-code">
+                {% trans "Return" %}:
+                <span data-bind="text: exit_code"></span>
+              </span>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    {% endif %}
+
+  </div>
+{% endblock %}
```

Comment:
Ci sono delle stringhe in inglese ma vedo che sono comprese in blocchi `{% blocktrans trimmed %} ... {% endblocktrans %}`